numpy/typing/tests/test_typing.py

```diff
@@ -123,6 +126,79 @@ def _test_fail(path: str, error: str, expected_error: Optional[str], lineno: int
         raise AssertionError(_FAIL_MSG2.format(lineno, expected_error, error))
 
 
+def _construct_format_dict():
+    dct = {k.split(".")[-1]: v.replace("numpy", "numpy.typing") for
+           k, v in _PRECISION_DICT.items()}
+
+    return {
+        "uint8": "numpy.unsignedinteger[numpy.typing._8Bit]",
+        "uint16": "numpy.unsignedinteger[numpy.typing._16Bit]",
+        "uint32": "numpy.unsignedinteger[numpy.typing._32Bit]",
+        "uint64": "numpy.unsignedinteger[numpy.typing._64Bit]",
+        "int8": "numpy.signedinteger[numpy.typing._8Bit]",
+        "int16": "numpy.signedinteger[numpy.typing._16Bit]",
+        "int32": "numpy.signedinteger[numpy.typing._32Bit]",
+        "int64": "numpy.signedinteger[numpy.typing._64Bit]",
+        "float16": "numpy.floating[numpy.typing._16Bit]",
+        "float32": "numpy.floating[numpy.typing._32Bit]",
+        "float64": "numpy.floating[numpy.typing._64Bit]",
+        "complex64": "numpy.complexfloating[numpy.typing._32Bit, numpy.typing._32Bit]",
+        "complex128": "numpy.complexfloating[numpy.typing._64Bit, numpy.typing._64Bit]",
+
+        "ubyte": f"numpy.unsignedinteger[{dct['_NBitByte']}]",
+        "ushort": f"numpy.unsignedinteger[{dct['_NBitShort']}]",
+        "uintc": f"numpy.unsignedinteger[{dct['_NBitIntC']}]",
+        "uintp": f"numpy.unsignedinteger[{dct['_NBitIntP']}]",
+        "uint": f"numpy.unsignedinteger[{dct['_NBitInt']}]",
+        "ulonglong": f"numpy.unsignedinteger[{dct['_NBitLongLong']}]",
+        "byte": f"numpy.signedinteger[{dct['_NBitByte']}]",
+        "short": f"numpy.signedinteger[{dct['_NBitShort']}]",
+        "intc": f"numpy.signedinteger[{dct['_NBitIntC']}]",
+        "intp": f"numpy.signedinteger[{dct['_NBitIntP']}]",
+        "int_": f"numpy.signedinteger[{dct['_NBitInt']}]",
+        "longlong": f"numpy.signedinteger[{dct['_NBitLongLong']}]",
+
+        "half": f"numpy.floating[{dct['_NBitHalf']}]",
+        "single": f"numpy.floating[{dct['_NBitSingle']}]",
+        "double": f"numpy.floating[{dct['_NBitDouble']}]",
+        "longdouble": f"numpy.floating[{dct['_NBitLongDouble']}]",
+        "csingle": f"numpy.complexfloating[{dct['_NBitSingle']}, {dct['_NBitSingle']}]",
+        "cdouble": f"numpy.complexfloating[{dct['_NBitDouble']}, {dct['_NBitDouble']}]",
+        "clongdouble": f"numpy.complexfloating[{dct['_NBitLongDouble']}, {dct['_NBitLongDouble']}]",
+
+        # numpy.typing
+        "_NBitInt": dct['_NBitInt'],
+    }
+
+
+#: A dictionary with all supported format keys (as keys)
+#: and matching values
+FORMAT_DICT: Dict[str, str] = _construct_format_dict()
+
+
+def _parse_reveals(file: IO[str]) -> List[str]:
```

Comment:
@mattip I'm not sure there is anything to expose here, unless we'd want to properly expose the entire `numpy.typing` tesing suite?

All this PR does is taking a set of comments and formatting them (when applicable), thus allowing us to use aliases.
It doesn't touch `mypy` itself or its output.